editor/editor_file_system.cpp

```diff
@@ -1674,6 +1676,8 @@ void EditorFileSystem::_update_scene_groups() {
 		if (!scene_groups.is_empty()) {
 			ProjectSettings::get_singleton()->add_scene_groups_cache(path, scene_groups);
 		}
+
+		ep.step(TTR("Updating Scene Groups..."), step_count++);
```

Comment:
You could display scene name here. Or exact progress (e.g. "(100 / 3050)").

Reply:
I like the idea but I feel it would be better if placed at the start of the line so that it doesn't move and maybe use space padding for the first number so that it stay the same size. I would also use `[` but not 100% sure if that isn't a personal preference.

Reply:
If we had https://github.com/godotengine/godot-proposals/issues/2399 it could be used for progress dialog.